compiler/rustc_error_messages/src/lib.rs

```diff
@@ -62,6 +62,7 @@ fluent_messages! {
     typeck => "../locales/en-US/typeck.ftl",
     mir_dataflow => "../locales/en-US/mir_dataflow.ftl",
     symbol_mangling => "../locales/en-US/symbol_mangling.ftl",
+    codegen_ssa => "../locales/en-US/codegen_ssa.ftl",
```

Comment:
Can you sort this list and add the new line at the appropriate point?

Reply:
Updated [here](https://github.com/rust-lang/rust/pull/102612/commits/8d94cda8da8e3350f972d5c7a06dae6dc90b4fdd)